src/applications/personalization/profile/components/personal-information/ContactInformationField.jsx

```diff
@@ -202,6 +202,7 @@ class ContactInformationField extends React.Component {
   justClosedModal(prevProps, props) {
     return (
       (prevProps.showEditView && !props.showEditView) ||
+      (prevProps.showRemoveModal && !props.showRemoveModal) ||
```

Comment:
The OPEN_MODAL action was still getting dispatched via the timeout after a successful deletion. This fixes that little quirk.